apps/mitt-konto/src/MittKonto/Main/Views/UserView/Subscription/Helpers.purs

```diff
@@ -55,14 +56,16 @@ translatePaymentMethod paymentMethod =
     DirectPayment        -> "Direktbetalning"
     UnknownPaymentMethod -> "Okänd"
 
-isPeriodExpired :: DateTime -> Maybe JSDate -> Boolean
-isPeriodExpired baseDate endDate =
+isPeriodExpired :: Boolean -> DateTime -> Maybe JSDate -> Boolean
+isPeriodExpired includeCurrentDay baseDateTime endDate =
   case endDate of
     -- If there's no end date, the period is ongoing
     Nothing   -> false
     Just date ->
-      let endDateTime = toDateTime date
-      in maybe true (_ < baseDate) endDateTime
+      let endDateTime = toDate date
+          baseDate = DateTime.date baseDateTime
+          op = if includeCurrentDay then (\a b -> a < b) else (\a b -> a <= b)
```

Comment:
Is the condition correct? If we want to include current day, shouldn't the check be `a  <= b`?

Also we can ditch the lambdas if it were:
```
op end = if includeCurrentDay then end < baseDate else end <= baseDate
maybe true op endDateTime
```

Reply:
I changed the variable name to `excludeCurrentDay` as that was what I meant.